lib/components/molecules/Toast/Toast.style.js

```diff
@@ -0,0 +1,77 @@
+import { css } from 'styled-components';
+
+const ToastStyles = css`
+  position: fixed;
+  display: flex;
+  align-items: center;
+  min-height: 80px;
+  min-width: 400px;
+  padding: 0 var(--spacing-3);
+  .toast-loader {
+    position: absolute;
+    bottom: 0;
+    height: 7px;
+    left: 0;
+    width: 100%;
+    background-color: var(--white);
+    opacity: 0.4;
+  }
+  ${props =>
+    props.placement === `top-left`
+      ? `top: 30px;left: 30px;animation: var(--animation-slide-right);`
+      : ``};
+  ${props =>
+    props.placement === `top-right`
+      ? `top: 30px;right: 30px;animation: var(--animation-slide-bottom);`
+      : ``};
+  ${props => (props.placement === `bottom-left` ? `bottom: 30px;left: 30px;` : ``)};
+  ${props =>
+    props.placement === `bottom-right`
+      ? `bottom: 30px;right: 30px;animation: var(--animation-slide-left);`
+      : ``};
+  ${props =>
+    props.placement === `center-center`
+      ? `top: calc(50% - 40px);left: calc(50% - 200px);animation: var(--animation-slide-up);`
+      : ``};
+  ${props => (props.placement === `top-center` ? `top: 30px;left: calc(50% - 200px);` : ``)};
```

Comment:
This is Too massive a CSS, not need to create so many variations. No one is going to use this CSS much anyway.